### DISCUSSION TITLE
graphical Hierarchy of models needed

### DISCUSSION THREAD

**Discussion by hazho** [wagtail/wagtail#9301] [General]
I've raised an issue, but I forgot to link it here as a discussion, please feel free to debate..!
Issue #9012 
 https://github.com/wagtail/wagtail/issues/9012

**Comment by UG-MAN** [comment]
Hello everyone,
I am Ugo, an Outreachy applicant.
I'm looking forward to making great contributions to Wagtail documentation.

Please where do I start from?

**Reply by hazho** [reply]
Welcome @UG-MAN , Why not trying the same question in the official slack channels?

**Reply by UG-MAN** [reply]
Thank you @hazho.

**Reply by hazho** [reply]
You are very welcome
